server/odc-service/src/main/java/com/oceanbase/odc/service/task/config/TaskFrameworkConfiguration.java

```diff
@@ -75,11 +73,10 @@ public JasyptEncryptorConfigProperties JasyptEncryptorConfigProperties(
 
     @Lazy
     @Bean("taskFrameworkSchedulerFactoryBean")
-    public SchedulerFactoryBean taskFrameworkSchedulerFactoryBean(DataSource dataSource,
```

Comment:
After switching to RAM mode, are there no concurrency issues with the existing tasks? I don't seem to see any locking logic implemented.

Reply:
daemon job fetch every task record by lock, see `StartPreparingJob#startJob`